tox.ini

```diff
@@ -8,13 +8,20 @@ minversion = 4.4
 
 [testenv]
 description = run tests (unit, unitcov, functional)
+passenv =
+    CMAKE_ARGS
 # Use PyTorch CPU build instead of CUDA build in test envs. CUDA dependencies
 # are huge. This reduces venv from 5.7 GB to 1.5 GB.
 setenv =
     PIP_EXTRA_INDEX_URL=https://download.pytorch.org/whl/cpu
+    CMAKE_ARGS={env:CMAKE_ARGS:-DLLAMA_NATIVE=off}
 package = wheel
 wheel_build_env = pkg
```

Comment:
I've noticed that `pkg` environment is built using `python3.12`. I am not sure it's an actual issue. (For what I observe, tests still pass.)

```
$ .tox/pkg/bin/python --version
Python 3.12.3
```

The following change seems to switch to 3.11:

```
--- a/tox.ini
+++ b/tox.ini
@@ -34,6 +34,9 @@ allowlist_externals =
 [testenv:py3]
 basepython = python3.11

+[testenv:pkg]
+basepython = {[testenv:py3]basepython}
+
 [testenv:py3-unit]
 basepython = {[testenv:py3]basepython}

```

Since you are clearly more knowledgeable about packaging / tox, I'd like to ask: do you anticipate any problems with us using py3.12 for wheel build environment? Or it's a non-issue?